about.md

```diff
@@ -33,3 +33,4 @@ On campus, FOSS@MAGIC does the following:
 * Partners with [RIT Linux Users Group](https://ritlug.com/) and [Women in Computing](http://wic.rit.edu/) organizations
 * Brings industry leaders to our FOSS Talks speaker series
 * Support student and faculty projects
+* Organizes [Future is Open conference](/events/2019/10/26/the-future-is-open) to showcase open source accomplishments of students, faculty, and alums
```

Comment:
Can't open this link! Otherwise, it looks good to me.

Reply:
@10eMyrT Good catch, but it's expected. :smile: It's an internal link that will end up being linked to this page when Jekyll processes it into HTML: https://fossrit.github.io/events/2019/10/26/the-future-is-open/